templates/openharmony/entry/src/main/ets/workers/cocos_worker.ts

```diff
@@ -57,11 +57,14 @@ uiPort._messageHandle = function(e) {
         case "onXCLoad":
             const renderContext = nativerender.getContext(ContextType.NATIVE_RENDER_API);
             renderContext.nativeEngineInit();
+
+            <% if(!useV8) { %>
             launchEngine().then(() => {
                 console.info('launch CC engine finished');
             }).catch(e => {
                 console.error('launch CC engine failed');
             });
+            <% } %>
```

Comment:
Also need to add control at `import { launchEngine } from '../cocos/game'`.
If it's in v8 mode, don't import game.ts

Reply:
updated